__tests__/lint/assertions-pattern-error/snapshot.js

```diff
@@ -5,7 +5,21 @@ exports[`E2E lint assertions-pattern-error 1`] = `
 validating /openapi.yaml...
 [1] openapi.yaml:20:16 at #/paths/~1pet~1findByStatus/get/summary
 
-Operation summary should have 'test' word
+summary-pattern assertion failed because the Operation summary didn't meet the assertions: "summary example" should match a regex /test/
```

Comment:
I'm not sure why it doesn't combine two assertion messages. Must something unrelated to this PR.

Reply:
@RomanHotsiy We will create a separate PR to fix it